### PR TITLE
Feature/DP-9521 MF Update location row on org and service pages

### PR DESCRIPTION
**Pull request by Tregonia**
## Description
Updating Location row to new theme

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-9521)

## Steps to Test
* Go here
  * https://mayflower.digital.mass.gov/?p=pages-organization
  * https://mayflower.digital.mass.gov/?p=pages-organization-elected-official
  * https://mayflower.digital.mass.gov/?p=pages-organization-boards
  * https://mayflower.digital.mass.gov/?p=pages-service
* Do this
  * Scroll to the location row (row with the map)
  * Review component agains updated Invision comps in ticket

## Screenshots
BackstopJS images are included in PR

### CODE REVIEW

**Comment by Tregonia** [comment]
I am not certain what else to do on this PR to make this pass. There are changes on Backstop that are in no way related to this PR; and some failed tests that have no changes at all.

File: patternlab/styleguide/source/assets/scss/03-organisms/_mapped-locations.scss [hunk 1]
```diff
@@ -1,59 +1,53 @@
 .ma__mapped-locations {
-  
-  .main-content--full .page-content > & {
```
Comment (clair0917) [inline patternlab/styleguide/source/assets/scss/03-organisms/_mapped-locations.scss]: this rule is usually only applied to the patternlab page, to keep the component from running up against the edges of the page. Could you double check it for me please?

**Review by clair0917 — APPROVED** [review]
One small comments, but it looks great

**Comment by edwardchan** [comment]
Will need to fix Backtop errors.

**Pull request merged**